devel/fortio/cmd/fortio/pingsrv.go

```diff
@@ -146,17 +145,16 @@ func healthCheck(serverAddr string, svcname string, n int) {
 	fmt.Printf("Statuses %v\n", statuses)
 }
 
-func main() {
-	flag.Parse()
-	if *hostFlag != "" {
-		// TODO doesn't work for ipv6 addrs etc
-		dest := fmt.Sprintf("%s:%d", *hostFlag, *portFlag)
-		if *doHealthFlag {
-			healthCheck(dest, *healthSvcFlag, *countFlag)
-		} else {
-			clientCall(dest, *countFlag, *payloadFlag)
-		}
+func grpcClient() {
+	if len(flag.Args()) != 1 {
```

Comment:
flag.Args is the remaining args after parsing the known flags. Anyway, it doesn't matter a ton: all of this stuff goes away with the move to cobra, which natively supports sub-commands and flags that are sub-command specific.